src/quo/components/text_combinations/channel_name/view.cljs

```diff
@@ -0,0 +1,40 @@
+(ns quo.components.text-combinations.channel-name.view
+  (:require [quo.components.icon :as icon]
+            [quo.components.markdown.text :as text]
+            [quo.components.text-combinations.channel-name.style :as style]
+            [quo.theme]
+            [react-native.core :as rn]))
+
+(defn icons
+  [{:keys [theme unlocked? muted? blur?]}]
+  [rn/view {:style style/icons-container}
+   (when unlocked?
+     [rn/view
+      {:style               style/icon
+       :accessibility-label :channel-name-unlocked-icon}
+      [icon/icon :i/unlocked
+       {:color (style/unlocked-icon-color theme blur?)
+        :size  20}]])
+
+   (when (and unlocked? muted?)
+     [rn/view {:style style/icons-gap}])
+
+   (when muted?
+     [rn/view
+      {:style               style/icon
+       :accessibility-label :channel-name-muted-icon}
+      [icon/icon :i/muted
+       {:color (style/muted-icon-color theme blur?)
+        :size  20}]])])
+
+(defn- view-internal
+  [{:keys [unlocked? muted? channel-name] :as props}]
+  [rn/view {:style style/container}
+   [text/text
+    {:size   :heading-1
+     :weight :semi-bold}
+    (str "# " channel-name)]
```

Comment:
Wondering if we should put the hashtag symbol at the end for RTL languages. We should revisit this in the future when we start adding new languages to the app.

Reply:
Yes, interesting.

All the channel names should change if so 🤔 I'm wondering the same.